docker-compose.commercial.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  index:
+    extends:
+      file: docker-compose.yml
+      service: index
+
+  app:
+    extends:
+      file: docker-compose.yml
+      service: app
+    environment:
+      YENTE_MANIFEST: "manifests/commercial.yml"
+      OPENSANCTIONS_DELIVERY_TOKEN: "your-secret-token"
```

Comment:
oh that's clever - using the environment variable so the file doesn't have to be mounted in.

when extending, is `environment` cumulative, or will this overwrite `environment in the original, meaning the YENTE_INDEX_URL setting gets lost?